src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="1"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__image"
+        src="./images/imac.jpeg"
+        alt = "imac"
+        >
+      <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars_star"></div>
```

Comment:
Change the class name 'stars_star' to 'stars__star' to follow the BEM naming convention.